docs/content/media.md

```diff
@@ -1,52 +1,59 @@
 # Media
 
-Media are only created by adding them to an Item. Media cannot exist independently. 
+Omeka S accepts most files and file types, and can be customized to accept or reject file types of your choice. You may wish to format your multimedia files according to what can best be embedded and streamed in modern browsers; see the [Media file types section at the end of this page](#media-file-types). If you are having difficulty or are seeing file-validation errors, you can adjust the accepted file types and extensions in the [Allowed media and files options of the Security Settings](../admin/settings.md#security).
+
+Media are only created by adding them to an item; media cannot exist independently. If you wish to upload files unattached to items, see the [Assets page](../admin/assets.md). 
 
 To view the media associated with an item, click on the media’s name in the right-hand sidebar on the item view page.
 
-To view all media in the installation, navigate to the Items browse first. The media browse will appear below Items in the left-hand navigation.
+To view all media in the installation, navigate to the **Items** tab first. The **Media** tab will appear below Items in the left-hand navigation.
```

Comment:
I maybe repetitively noting this: wasn't sure where we landed on "tab."

Reply:
I missed this one.